packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -238,13 +300,9 @@ class MessageList extends Widget<Properties> {
         super._optionChanged(args);
     }
   }
-
-  update(): void {
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    this._scrollable.update();
-
-    this._scrollContentToLastMessage();
-  }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// (MessageList as any).include(DataHelperMixin);
```

Comment:
Redundant comments?

Reply:
Thanks. Yep, this part from other task

Reply:
https://github.com/DevExpress/DevExtreme/pull/28111/commits/4b842e1b19494eb9baf6a8a204fa91b23a683b14